docs/bundles/SyliusRbacBundle/index.rst

```diff
@@ -1,7 +1,7 @@
 SyliusRbacBundle
 ================
 
-Role-Based-Access-Control for Symfony2 applications, integrated with Doctrine ORM. Managable via UI.
+Role-Based-Access-Control for Symfony applications, integrated with Doctrine ORM. Managable via UI.
```

Comment:
Thanks @dantleech for these remarks! :D although I'm not sure if I should change it in this PR. :D